### PR TITLE
Fix max fetch

### PR DESCRIPTION
**Pull request by A3a3e1**

### CODE REVIEW

**Comment by ShkarupaNick** [comment]
Please upddate package.json

File: README.md [hunk 1]
```diff
@@ -341,6 +341,8 @@ You can find more detail information in the [Platform Events Intro Documentation
 
 2. `LOG_LEVEL` - `trace` | `debug` | `info` | `warning` | `error` controls logger level
 
+3. `MAX_FETCH_COUNT` - limit for a number of messages that can be fetched for triggers and actions. 1,000 is the default value when the variable is not set.
```
Comment (stas-fomenko) [inline README.md:344]: need to specify actions/triggers that are could be affected by this env

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+## 1.3.3 (May 5, 2020)
+
+* Fix bug with 1,000 objects limit in Query, Lookup Object, Lookup Objects actions, Get New and Updated Objects Polling and Query triggers
```
Comment (stas-fomenko) [inline CHANGELOG.md:3]: add also that envar has been added

File: README.md [hunk 3]
```diff
@@ -49,6 +49,9 @@ You can get error `refresh token has been expired` if the same user account was
 Executing a SOQL Query that may return many objects. Each resulting object is emitted one-by-one. Use the Salesforce Object Query Language (SOQL) to search your organization’s Salesforce data for specific information. SOQL is similar to the SELECT statement in the widely used Structured Query Language (SQL) but is designed specifically for Salesforce data. This action allows you to interact with your data using SOQL.
 Empty object will be returned, if query doesn't find any data.
 
+Please pay attention to the environment variable `MAX_FETCH_COUNT` to configure the action correctly (see [Environment variables section for the details](#environment-variables) for the details).
+Be careful! For 'Query' action Salesforce platform limit is 2000!
```
Comment (A3a3e1) [inline README.md:53]:
```suggestion
Be careful! Salesforce sets limit for the 'Query' action to 2000 objects to fetch at once!
```

**Pull request merged**